t/pmbp-git/submodule-add-r-1.t

```diff
@@ -23,7 +23,4 @@ cd $tempdir/hoge/git1 && git init && touch b && git add b && git commit -m new
 (cat $tempdir/foo/modules/git1/b && echo "not ok 6") || echo "ok 6"
 (cat $tempdir/foo/modules/git1.2/b && echo "ok 7") || echo "not ok 7"
 
-(perl $pmbp --root-dir-name $tempdir/foo --add-git-submodule-recursively aa/m\ $tempdir/git1 && echo "ok 8") || echo "not ok 8"
```

Comment:
Because this test case is tested on t/pmbp-git/submodule-add-r-3-parent-dir.t, it was deleted.